src/App.tsx

```diff
@@ -1,24 +1,125 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { createTodo, getTodos, deleteTodo } from './api/todos';
+import { Footer } from './components/footer_filter/footer_filter';
+import { Header } from './components/header_input/header_input';
+import { Main } from './components/main_todos-list/main_todos-list';
 
-const USER_ID = 0;
+const USER_ID = 10548;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [input, setInput] = useState('');
+  const [status, setStatus] = useState('all');
+  const [addTodo, setAddTodo] = useState('');
+  const [removeTodo, setRemoveTodo] = useState(0);
+
+  async function loadedTodos(f: (USER_ID: number) => Promise<Todo[]>) {
+    const result = await f(USER_ID);
+
+    setTodos(result);
+  }
+
+  useEffect(() => {
+    loadedTodos(getTodos);
+  }, [!addTodo]);
+
+  const handleChangeInput = (event : React.ChangeEvent<HTMLInputElement>) => {
+    setInput(event.target.value);
+  };
+
+  const handleStatus = (
+    event: React.MouseEvent<HTMLAnchorElement, MouseEvent>,
+  ) => {
+    setStatus(event.currentTarget.type);
+  };
+
+  const handleAddTodo = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    if (input.trim().length > 0) {
+      setAddTodo(input);
+    }
```

Comment:
here should be createTodo calling which response new todo 
and after response update todos in state (set new todo to array)
```suggestion
 
```